src/select-list-view.js

```diff
@@ -37,6 +37,14 @@ module.exports = class SelectListView {
     this.refs.queryEditor.setText('')
   }
 
+  setQueryText (value, isSelect = false) {
```

Comment:
ah ok, I got it you mean when the method called. I would stick to just boolean param, developer anyway have to know the API which he use. I also see similar signature already used in atom-select-list:
`selectIndex (index, updateComponent = true)`